Sources/Plasma/PubUtilLib/plSDL/plSDL.h

```diff
@@ -188,8 +193,9 @@ class plStateChangeNotifier
 public:
     plStateChangeNotifier();
     plStateChangeNotifier(float i, plKey k);
+    virtual ~plStateChangeNotifier();
```

Comment:
```suggestion
    virtual ~plStateChangeNotifier() = default;
```

Reply:
Looks like I did this because `plKey` is only forward-declared in the header.